synapseclient/__main__.py

```diff
@@ -289,6 +291,18 @@ def onweb(args, syn):
 
 def setProvenance(args, syn):
     """Set provenance information on a synapse entity."""
+    if isinstance(args.id, str) and os.path.isfile(args.id):
+        results = syn.restGET('/entity/md5/%s' % utils.md5_for_file(args.id).hexdigest())['results']
+        if args.limitSearch is not None:
+            results = filter_id_by_limitSearch(syn, results, args.limitSearch)
+
+        set_id_result = check_id_results(results)
+        if len(set_id_result) > 1:
+            raise SynapseProvenanceError('There are more than one identical content for this file in different '
```

Comment:
Include the ids of the matching files to help users debug? e.g.
`f"{args.id} matched more than one file: {list of ids}"`